lib/Sema/TypeCheckAttr.cpp

```diff
@@ -2102,6 +2105,27 @@ void AttributeChecker::visitImplementsAttr(ImplementsAttr *attr) {
   }
 }
 
+void AttributeChecker::visitFrozenAttr(FrozenAttr *attr) {
+  switch (D->getModuleContext()->getResilienceStrategy()) {
+  case ResilienceStrategy::Default:
+    // FIXME: Don't warn about the use of this attribute in the standard
```

Comment:
You might be able to get away with removing this now that resilience is on

Reply:
Ah, yes, I forgot about this!